modules/nf-core/bakta/baktadbdownload/main.nf

```diff
@@ -1,35 +1,33 @@
 process BAKTA_BAKTADBDOWNLOAD {
     label 'process_single'
 
-    conda "bioconda::bakta=1.6.1"
+    conda "bioconda::bakta=1.7.0"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/bakta:1.6.1--pyhdfd78af_0' :
-        'quay.io/biocontainers/bakta:1.6.1--pyhdfd78af_0' }"
+        'https://depot.galaxyproject.org/singularity/bakta:1.7.0--pyhdfd78af_1' :
+        'quay.io/biocontainers/bakta:1.7.0--pyhdfd78af_1' }"
 
     output:
-    path "db/"              , emit: db
+    path "db*"              , emit: db
     path "versions.yml"     , emit: versions
 
     when:
     task.ext.when == null || task.ext.when
 
     script:
     def args = task.ext.args ?: ''
-
     """
     bakta_db \\
         download \\
         $args
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        bakta: \$(echo \$(bakta_db --help) 2>&1 | sed 's/.*Version: //g;s/ DOI.*//g')
+        bakta: \$(echo \$(bakta_db --version) 2>&1 | cut -f '2' -d ' ')
     END_VERSIONS
     """
 
     stub:
     def args = task.ext.args ?: ''
-
     """
     echo "bakta_db \\
```

Comment:
does this generate anything like the expected output `path "db*" , emit: db` ? at first glance, you need something like
```bash
touch dbfake.file
```
or maybe
```
mkdir -p db/fake
touch db/fake/empty.txt
```

Reply:
if I'm wrong, then also consider if you need a closed parenthesis:
```bash
echo "bakta_db download"
```

Reply:
It should be fine to just `mkdir` the directory `db` which is the expected input for the bakta tool (i.e. mirroring the behaviour of the normal bakta database download). Whatever is in that folder doesn't matter for the stub here ;)

The closed parenthesis comment I don't understand, `echo` doesn't need parentheses, right? Or do you mean "quotation marks"? In that case, it is opened in line 32 and closed in 34.